### ISSUE TITLE
HW1 races performance problems

### ISSUE THREAD

**Issue by Echo1707** [HW-PlayersPatch/2.3#130]
Thanks to Robey for pushing me and for his pointers, I'm publishing this spreadsheet about research costs and times for various factions. While reviewing it, I realized I made some errors. I can't explude the possibility that there are some left, so please give it an attentive look before we start commenting about it's content.

https://www.dropbox.com/s/p2640n5c9rjw3l9/Technologies.xlsx?dl=0

All times for hw1 researches are with 1 research ship. Do you know the formula to calculate times with 2-3-4-5-6 research ships?

**Comment by cloaked2222** [comment]
All of this info and more is accurately documented in the Stats Tool. See the 2.3\Resources folder.

**Comment by Echo1707** [comment]
The point of that file is to show in a nice way that HW1 ions, CC, DD and HC researches cost too much compared to the other races. Looking at the sheet it might not seem so for HW1 ion frigs, but that's just because I only took into account 1 research ship. There is definitively the time to build more.

**Comment by cloaked2222** [comment]
Look at the Core Comparison files, for a comprehensive comparison off all the costs involved (research mods/ships, research, upgrades, ship costs, etc). HW1 and HW2 are very well balanced cost and time wise in b9. I spent many weeks fine tuning this, so you'll be very hard pressed to find any significant flaws in the design. Its mainly a matter of studying/understanding it.


Generally hw1 ship build time is slower than hw2 as hw1 has 4 production ques while hw2 only has 2 production ques.


You can also look at the HW1 Design excel file for additional details on HW1/HW2 balance.


________________________________

**Comment by tom-mindlab** [comment]
Clearly something crucial has been missed here, I'm not going to pretend like I have the insight to be able to pick out exactly where, but HW1 is really unpleasant to play now - especially in 1v1s, the research costs are astronomically expensive. The primary offender is absolutely corvettes, which cost absurd amounts of money and time to field. This really is a shame because this release has a lot of great changes which are completely mired by these cost-alignment ideas.

There's also general dismay in regards to the resilience of missile corvettes (hw2 vettes in general but mostly missiles). They're absolutely the best unit in the game right now in my eyes and many players are resolute vaygr mains due to how abusable this unit is - there really isn't a way to prevent them from killing all your collectors and probably not even dying due to dock mechanics.

HW players are pretty bad at innovating so I'll try my best to scope something out for HW1, however at a glace I feel like this race really isn't worth playing right now - perhaps in 3v3s their repair and general utility are invaluable but elsewhere my kneejerk reaction is that this has been a step in the opposite direction for HW1

**Comment by Novaras** [comment]
I'm well aware that these times and costs have been carefully calculated to line up across the races, however this idea is founded on the notion that units across races are roughly equivalent in value, which is just not true whatsoever. HW2 strike are unquestionably stronger than HW1 strike simply because they are capable of docking. HW1 frigates are technically superior stat-wise, but inferior production wise - on the capital front, HW1 is outgunned and outproduced both.

I'd like to especially draw attention to the cost of getting out a heavy cruiser - a unit which can't support itself at all and is weaker than a BC 1-1. Pretty sure it's been spreadsheeted, but the price hike is almost comically huge.

The biggest flop of all is the corvette price hikes. Why? Why was this done? How much playtesting was performed before it was shipped as ready? HW1 corvettes were the bread-and-butter of the race - the price for *light vette tech* is 1300 damn RUs! I'm sorry to be so abrasive here but this is really a HUGE fuckup - there's no silver lining for this unit class.

**Comment by Novaras** [comment]
Unit testing and cost calculation should have been the precursor to extensive playtesting, not the release phase. In real-game scenarios is matters *immensely* that frigates struggle to prevent squadron docks. It matters *immensely* that HW2 frigates will beat-out HW1 frigates in a production battle. This version went so far and dropped the ball hard at the final hurdle, which is depressing to see.

Sorry to be so critical, this mod is still great and a ton of fun - using new defense fields has been a blast and seeing destroyers grace the field after so long is refreshing. As I mentioned earlier, there's plenty of great content here, but there are some crippling oversights and oversimplifications that have been made which sully the quality of this release.

**Comment by Novaras** [comment]
I guess I should highlight that my gripes here are about the state of the HW1 races - HW2 seems to be in a pretty good place, finally having fun in this MU

**Comment by Novaras** [comment]
Will do.

TL;DR of below:
- HW1 strike are severely outclassed by docking squadrons
  - HW1 repairers are nice for frigates+, but are usually very weak with corvettes
  - Repair vettes die too easily
- HW1's stacked tech means a lot of dead RUs
  - HW2 has many critical units at base tech, especially carriers
  - HW1 routinely has to decide between two critical units (ex: gravwell now vs carrier now)
- HW2 corvettes are too durable and corvettes are in general a weakness for HW1
  - Missile corvettes are out of control and borderline cannot be killed if controlled well.

---

Repairers are great although they are definitely weakest when helping corvettes as they struggle to get to the scene in time - repair vettes are also paper mache and naturally targeted by opponents. HW1 strike are *severaly* outclassed by HW2 as they simply cannot overcome the benefits of being able to dock.

I've been playing quite a bit since a few days back on 2.3; the times I've observed to get a fighting force out of vettes or capitals is truly monstrous and even fighters feel difficult. Frigates are in a better place than before although HW1 frigates cannot handle fighters at all (drones still really struggle) and you generally don't have the time of day to fork your tech to grab gravwells thus delaying carrier tech even further - not to mention you're already strapped for cash trying to afford it all.

HW1's stacked tech tree is really such a huge issue for them - maybe the costs of getting x unit out are the same for taii as they are for vay, however vay can tech *directly to this unit*, whereas hw1 need to traverse quite a lot of 'useless' tech to reach what they want. In addition, the fact that you can only research one tech at a time turns out to hurt HW1 far more than it hurts HW2 - HW2 truly have all their critical units at base tech (no research), this is especially true for carriers which hiig can get out easily and vay can get out comically easy. Keep in mind that HW2 can stack research modules for faster teching if they want - and *sell* those module after. Vaygr is especially hilarious with this.

Selling modules is also a huge bonus for good HW2 players. Modules themselves are, in practice, not a liability at all as it often takes just as much ru/time/supply investment killing them as it took to build them. Being able to *sell* a capship facility after getting the first CC out is a huge bonus. This is also famously true for the hgn adv. research module which is probably the most sold of all subsystems (vay carrier facilities notwithstanding).

BTW, was torp missile speed increased after all? I've seen collectors outpacing them again.

**Comment by cloaked2222** [comment]
With so many corvette stats changing i needed one ship to stay the same as sort of a reference point to compare b7 and b8. That ship was the hiig torp, as hw1 ass frigs were significantly changed. See here for some good ideas that didnt quite make b8, but maybe good for b10:
https://github.com/HW-PlayersPatch/2.3/issues/83
